backend/views/syson-diagram-interconnection-view/src/main/java/org/eclipse/syson/diagram/interconnection/view/nodes/RootNodeDescriptionProvider.java

```diff
@@ -137,7 +137,7 @@ private ConditionalNodeStyle createPartUsageConditionalNodeStyle() {
 
     private NodePalette createNodePalette(IViewDiagramElementFinder cache) {
         var callEditService = this.viewBuilderHelper.newChangeContext()
-                .expression(AQLConstants.AQL_SELF + ".directEdit(newLabel)");
+                .expression(AQLConstants.AQL_SELF + ".directEditNode(newLabel)");
```

Comment:
Could you please change it to `AQLUtils.getSelfServiceCallExpression("directEditNode", "newLabel")`?

Reply:
Done